auxx/src/Mode.hs

```diff
@@ -213,11 +212,8 @@ instance (HasConfiguration, HasInfraConfiguration, HasSscConfiguration, HasCompi
     getLocalHistory = getLocalHistoryDefault
     saveTx = saveTxDefault
 
-instance MonadKnownPeers AuxxMode where
-    updatePeersBucket = realModeToAuxx ... updatePeersBucket
-
 instance MonadFormatPeers AuxxMode where
-    formatKnownPeers = OQ.Reader.formatKnownPeersReader (rmcOutboundQ . acRealModeContext)
+    formatKnownPeers formatter = withReaderT acRealModeContext (formatKnownPeers formatter)
```

Comment:
`MonadFormatPeers` will go away (for now the `RealMode` instance gives `pure Nothing`). Better solution is to have some debug report from the diffusion layer, because the set of known peers surely isn't the gold standard of trace information. https://iohk.myjetbrains.com/youtrack/issue/CSL-2109

Reply:
`withReaderT acRealModeContext` is `realModeToAuxx`, but since it will go away, it's not essential.